nepserver.py

```diff
@@ -1,13 +1,43 @@
 #!/usr/bin/env python3
+import json
+import socketserver
 from http.server import BaseHTTPRequestHandler, HTTPServer
 from datetime import datetime, timezone, timedelta
 from models import Datapoint
+from dnslib import DNSRecord, DNSHeader, RR, QTYPE, A
 from os import environ
-import json
+import threading
+import socket
 
 # A dictionary to hold the last values received from POST requests.
 last_values = {}
 
+# Environment Variables for DNS Configuration
+INTERCEPT_DOMAIN = 'www.nepviewer.net.'
+RESPONSE_IP = environ.get('RESPONSE_IP', '0.0.0.0')  # IP to respond with for intercepted domain
+FORWARD_DNS = environ.get('FORWARD_DNS', '8.8.8.8')    # DNS server to forward non-intercepted queries
+
+class DNSRequestHandler(socketserver.BaseRequestHandler):
+    def handle(self):
+        data, socket = self.request
+        request = DNSRecord.parse(data)
+
+        reply = DNSRecord(DNSHeader(id=request.header.id, qr=1, aa=1, ra=1), q=request.q)
+
+        if str(request.q.qname) == INTERCEPT_DOMAIN:
+            # Intercept specific domain and respond with specified IP
+            reply.add_answer(RR(request.q.qname, QTYPE.A, rdata=A(RESPONSE_IP), ttl=60))
+        else:
```

Comment:
make it possible to disable Forward
```suggestion
        else if FORWARD_DNS not in ["", "false", "False"]:
```